### ISSUE TITLE
Wide character in print at /usr/local/share/perl/5.22.1/Test2/Formatter/TAP.pm

### ISSUE THREAD

**Issue by ivanych** [pjlsergeant/test-bdd-cucumber-perl#125]
Hi Peter!

I wrote a feature file in Russian (Muiti byte string). And then pherkin command issued then print out ”Wide character in print”.

**test.feature**:
```
Feature: Тест
    тест тест тест

 Scenario: выполнить тест
    Given тест
```

**step_definitions/test.steps**:
```
#!perl

use strict;
use warnings;
use utf8;

use Test::More;
use Test::BDD::Cucumber::StepFile;

Given qr/тест/, sub {
    ok(1, 'Test');
};
```

**result**:
```
$ pherkin

  Тест
    тест тест тест

    Scenario: выполнить тест
Wide character in print at /usr/local/share/perl/5.22.1/Test2/Formatter/TAP.pm line 113.
      Given тест
```

**Comment by pjlsergeant** [comment]
Thanks Mikhail, I'll take a look into this

**Comment by pjlsergeant** [comment]
Hi @ivanych 

I can't replicate this on my machine. I wonder if you have an older version of Test::More? Could you try upgrading and see if that fixes it?

**Comment by ivanych** [comment]
```
$ perl -e 'use Test::More; print $Test::More::VERSION'
1.302120
```

```
pherkin 

  Тест
    тест тест тест

    Scenario: выполнить тест
Wide character in print at /usr/local/share/perl/5.22.1/Test2/Formatter/TAP.pm line 144.
      Given тест
```
Can you check it in my example? - https://github.com/ivanych/pherkintest

**Comment by ivanych** [comment]
It would be great to fix these warnings. I tried to do this and I got it - the warnings disappeared.

Here is my pull-request - #126

**Comment by pjlsergeant** [comment]
Fixed in 0.54
